packages/discord.js/src/managers/GuildForumThreadManager.js

```diff
@@ -20,7 +20,8 @@ class GuildForumThreadManager extends ThreadManager {
    * @typedef {BaseMessageOptions} GuildForumThreadMessageCreateOptions
    * @property {StickerResolvable} [stickers] The stickers to send with the message
    * @property {BitFieldResolvable} [flags] The flags to send with the message
-   * <info>Only `MessageFlags.SuppressEmbeds` and `MessageFlags.SuppressNotifications` can be set.</info>
+   * <info>Only `MessageFlags.SuppressEmbeds`, `MessageFlags.SuppressNotifications`, and `MessageFlags.IsVoiceMessage`
```

Comment:
It's still not documented so, or I'm missing something?
https://github.com/discord/discord-api-docs/blob/12d64791a79503e1172d6972a52e31c08cfe11cd/docs/resources/Message.md?plain=1#L761

Reply:
- https://github.com/discord/discord-api-docs/pull/7267